plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/S3RepositoryThirdPartyTests.java

```diff
@@ -56,7 +60,7 @@ protected SecureSettings credentials() {
     protected void createRepository(String repoName) {
         Settings.Builder settings = Settings.builder()
             .put("bucket", System.getProperty("test.s3.bucket"))
-            .put("base_path", System.getProperty("test.s3.base", "/"));
+            .put("base_path", System.getProperty("test.s3.base", "testpath"));
```

Comment:
Should we randomly use an empty `base_path` here?

Reply:
Currently it doesn't do anything because the tests actually set a base path, I just set it like this for my own private testing against real S3. I think this would require some additional work on the Gradle side.
Maybe do that in a follow up, I wanted to do some cleanups on the Gradle code upstream anyway? :)

Reply:
ok, we should definitely test this as that will often times be the default configuration. OK to do in a follow-up.